charts/nri-bundle/Chart.yaml

```diff
@@ -45,13 +45,21 @@ dependencies:
     version: 1.0.0
 
   - name: kube-state-metrics
-    version: 4.23.0
+    version: 4.25.0
     condition: ksm.enabled,kube-state-metrics.enabled
     repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics
+
+  - name: kube-state-metrics
+    version: 4.23.0
+    condition: ksm_v26.enabled,kube-state-metrics-v26.enabled
+    repository: https://prometheus-community.github.io/helm-charts
+    alias: kube-state-metrics-v26
```

Comment:
Have we checked if using an alias changes the KSM `app.kubernetes.io/name` label? If these labels change we should update the labelSelector `nri-kubernetes` uses to find KSM.

This is hard to do from `values.yaml`. I would suggest to not use aliases to avoid this, and have a safety check in `NOTES.txt` preventing the user from setting both dependencies at the same time.

Reply:
Here you have an example: https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/templates/NOTES.txt#L125-L131